src/domains/chart/chart-types.ts

```diff
@@ -42,8 +42,9 @@ export interface EasyPieChartData extends ChartDataBase {
 
 export type ChartData = DygraphData | EasyPieChartData | D3pieChartData
 
-interface Dimension {
+export interface Dimension {
   name: string
+  correlationScore?: Number
```

Comment:
I'm leaving this to be optionally used in charts metadata. (Attribute use just a string[])